sample-code/README.md

```diff
@@ -162,6 +162,20 @@ Use `buildAzureContentFilter()` to build the content filter.
 
 Send chat completion request with a custom header as the custom request configuration.
 
+#### Chat Completion Streaming
+
+`GET /orchestration-stream/chat-completion-stream`
```

Comment:
```suggestion
`POST /orchestration-stream/chat-completion-stream`
```
[req] Also add a payload example here. Would be nice to have to have a curl cmd here.

Reply:
[req] `/orchestration-stream/chat-completion-stream-json` missing.